docs/azure_arc_jumpstart/azure_arc_servers/scaled_deployment/configuration_manager_scaled_runscript/_index.md

```diff
@@ -0,0 +1,150 @@
+    ```
+
+    For example:
+
+    ```shell
+    az ad sp create-for-rbac -n "http://AzureArcServers" --role contributor
+    ```
+
+    Output should look like this:
+
+    ```json
+    {
+    "appId": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "displayName": "AzureArcServers",
+    "name": "http://AzureArcServers",
+    "password": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "tenant": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+    }
+    ```
+
+    > **Note: The Jumpstart scenarios are designed with as much ease of use in-mind and adhering to security-related best practices whenever possible. It is optional but highly recommended to scope the service principal to a specific [Azure subscription and resource group](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest) as well considering using a [less privileged service principal account](https://docs.microsoft.com/en-us/azure/role-based-access-control/best-practices)**
+
+* Create a new Azure resource group where you want your machine(s) to show up.
+
+    ![Screenshot showing Azure Portal with empty resource group](./01.png)
+
+* Download the [az_connect_win](https://github.com/microsoft/azure_arc/blob/main/azure_arc_servers_jumpstart/scripts/az_connect_win.ps1) PowerShell script.
+
+* Change the environment variables according to your environment and save the script.
+
+    ![Screenshot showing PowerShell script](./02.png)
+
+## Setting the PowerShell execution policy in Configuration Manager
+
+In order for Configuration Manager to run the PowerShell script in this scenario, we will need to set the PowerShell execution policy and allow the script authors to approve their own scripts.
+
+   > **Note: The Jumpstart scenarios are designed with as much ease of use in-mind and adhering to security-related best practices whenever possible. It is recommended to follow the security recommendations in your organization when it comes to PowerShell scripts execution policy and approvals**
+
+* Login to the Configuration Manager console.
+
+* After logging in, go to the “Administration” workspace. Under “Site Configuration”, select “Client Settings” and open the properties of the "Default Client Settings".
+
+   > **Note: You might have different client settings, choose the appropriate one that is deployed to the servers you are onboarding to Azure Arc**
+
+    ![Screenshot showing client settings in the configuration manager console](./03.png)
+
+* Click the “Computer Agent” settings, change the “PowerShell execution policy” to “Bypass”.
+
+    ![Screenshot showing Computer Agent settings in the configuration manager console](./04.png)
+
+## Configuration of the PowerShell script approval settings in Configuration Manager
+
+* Go to the “Administration” workspace. Under “Site Configuration”, select “Sites” and select the Configuration Manager site.
+* Click on “Hierarchy Settings”.
+    
+    ![Screenshot showing sites settings in the configuration manager console](./05.png)
```

Comment:
Adding steps numbers in the screenshot can help to better understand the screen flow